src/components/apps/toolbar/Toolbar.js

```diff
@@ -192,6 +185,20 @@ function AppsToolbar(props) {
                         />
                     )}
                 </Hidden>
+                <Hidden smDown>
+                    <Link href={`${NavigationConstants.TOOLS}`}>
+                        <Button
+                            id={build(appsToolbarId, ids.TOOLS_BTN)}
+                            className={classes.toolbarItems}
+                            variant="outlined"
+                            disableElevation
+                            color="primary"
+                            startIcon={<Build />}
+                        >
+                            {t("manageTools")}
+                        </Button>
+                    </Link>
```

Comment:
Is there no way to get to the tools view on mobile? 🤔

Reply:
Good Catch 👍🏽